projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -127,6 +149,12 @@ export class GoTableComponent implements OnInit, OnChanges, AfterViewInit {
     this.showTable = Boolean(this.tableConfig);
   }
 
+  setupConfigService(): void {
+    this.subscriptions.add(this.goConfigService.config.subscribe((config: GoConfigInterface) => {
+      this.brandColor = config.brandColor;
+    }));
+  }
```

Comment:
This is where we `subscribe` to the `GoConfigService` and assign a local `brandColor` variable to that of the current configs.

Reply:
I probably should have pointed this out to you earlier, but in [this PR](https://github.com/mobi/goponents/pull/693) I introduced a new method for unsubscribing when components are destroyed that utilizes the `takeUntil` RxJs operator.

So in this case, since this component has already been set up with that stuff, all you would have to do is change this to:
```javascript
this.goConfigService.config
  .pipe(takeUntil(this.destroy$))
  .subscribe((config: GoConfigInterface) => {
      this.brandColor = config.brandColor;
  });
```

And so you could remove `subscriptions: Subscription = new Subscription();` above.